src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +59,12 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList
+                .stream()
+                .filter(person -> person.getSex().equals(Person.Sex.MAN)
+                        && person.getAge() >= fromAge
+                        && person.getAge() <= toAge)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
Be sure to make it everywhere.

Reply:
fixed